server/config/template.json

```diff
@@ -2,5 +2,6 @@
   "SENDGRID_API_KEY": "API key for sendgrid",
   "SENDGRID_FROM_EMAIL": "address to send emails from",
   "SENDGRID_ENABLE": "should be true or false (boolean value)",
+  "DB_URI": "mongodb URI",
```

Comment:
This is good practice but since we have a mongo container running in docker it might be ok to hardcode it too

Reply:
Where do I get the URI for our mongo instance?

Reply:
It should just be `mongo://mongodb`. `mongodb` is the name of the container in docker-compose

Reply:
Or `mongo://mongodb/dbname`